src/components/pages/Publications/Publications.tsx

```diff
@@ -1,92 +1,175 @@
-        slidesToShow: 1,
-        slidesToScroll: 1,
-        infinite: true,
-        dots: true
-      }
-    }
-  ]
-};
+                <h2 id={PAGE_HASHES.Publications.ResearchArticles}>
+                    {Translate('ResearchArticles')}
+                </h2>
+                <p>
+                    {Translate("PublicationDescriptions", ["ResearchArticles"])}
+                </p>
+                {getPublications("articles")}
 
-export type PublicationsType = 'articles' | 'reports' | 'media' | 'biblioDigests';
+                <h2 id={PAGE_HASHES.Publications.PrivateSectorReports}>
+                    {Translate('PrivateSectorReports')}
+                </h2>
+                <p>
+                    {Translate("PublicationDescriptions", ["PrivateSectorReports"])}
+                </p>
+                {getPublications("reports")}
 
-const getCarouselOfPublicationsCards = (type: PublicationsType) => {
-  return (
-    <div className="publications-slider-container">
-      <Slider 
-        {...sliderSettings}
-      >
-          {
-            PublicationsInfo[type].map((publicationsProps, idx) => {
-              return <PublicationsCard
+                <h2 id={PAGE_HASHES.Publications.BiblioDigests}>
+                    {Translate('BiblioDigests')}
+                </h2>
+                <p>
+                    {Translate("PublicationDescriptions", ["LiteratureUpdateReports"])}
+                </p>
+                {getLinksOfPublications("biblioDigests")}
+
+                <h2 id={PAGE_HASHES.Publications.MonthlyReports}>
+                    {"Monthly Reports"}
+                </h2>
+                <p>
+                    {Translate("PublicationDescriptions", ["MonthlyReports"])}
+                </p>
+                {getLinksOfPublications("monthlyReports")}
+
+                <h2 id={PAGE_HASHES.Publications.MediaMentions}>
+                    {Translate('MediaMentions')}
+                </h2>
+                <p>
+                    {Translate("PublicationDescriptions", ["MediaMentions"])}
+                </p>
+                {getCarouselOfPublicationsCards("media")}
+            </div>
+        </div>
+    )
+}
+
+
+const getPublications = (type: PublicationsType) => {
+    return (
+        PublicationsInfo[type].map((publicationsProps) => {
+            return <PublicationsItem
                 day={publicationsProps.day ?? ""}
                 month={publicationsProps.month}
                 year={publicationsProps.year}
-                img={publicationsProps.img}
                 titleKey1={publicationsProps.titleKey1}
                 titleKey2={publicationsProps.titleKey2}
-                url={publicationsProps.url} />
-              })
-          }
-      </Slider>
-    </div>
-  )
+                url={publicationsProps.url}
+                authors={publicationsProps.authors}
+                publicationName={publicationsProps.publicationName}/>
+        }))
+}
+
+const sliderSettings = {

```

Comment:
Nit: Put this right beside the `getCarouselOfPublicationsCards` function because it is only used there now